src/utils/formatter.ts

```diff
@@ -4,6 +4,9 @@ import { Listing } from "networking/listings";
 // comma-separates terms and leaves no trailing commas
 export function formatAddress(...args: Array<string | undefined>): string {
   const clean = args.filter(str => !!str && !(str.toUpperCase() === 'US' || str.toUpperCase() === 'USA'));
+  if (clean.length < 3) {
+    return clean.join(', ');
+  }
```

Comment:
noticed some short addresses displayed as `SAN FRANCISCO CA` with no comma. This may be a band-aid for those small cases.

also not sure if `Vancouver, British Columbia, CAN` or `Vancouver, British Columbia CAN` is correct

Reply:
The way I tried to deal with this on the backend was:

    const clean = args.filter(str => !!str && !(str.toUpperCase() === 'US' || str.toUpperCase() === 'USA'));
    return /^([0-9]|-)+$/.test(clean[clean.length - 1]) ? // No comma before postal code, if present
      clean.slice(0, -1).join(', ') + ' ' + clean.slice(-1) :
      clean.join(', ');

...to use a regex to see if the last element looked like a postal code.

Reply:
wow nice. will use thx!